src/main/java/frc/robot/commands/WristCommand.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.subsystems.WristSubsystem;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class WristCommand extends Command {
+
+  private final WristSubsystem wristSubsystem;
+  private final double armDesiredPosition;
+  private final double wristDesiredPosition;
+
+  /** Creates a new WristCommand. */
+  public WristCommand(WristSubsystem wristSubsystem, double armAngle, double wristAngle) {
```

Comment:
Instead of passing in a raw armAngle and wristAngle as parameters, it'd be nice if I could pass in a mode instead.

The process would be something like this:
- Create an [enum](https://www.w3schools.com/java/java_enums.asp) for different states (GroundIntake, SourceIntake, CoralScore, Stowe). The enum would contain values for arm angle and wrist angle
- Modify the constructor to take in a WristSubsystem object and an enum value
- Set the wrist/arm angle based on the values of the enum

This would make the whole process more intuitive